json_serializable/lib/src/generator_helper.dart

```diff
@@ -128,7 +128,7 @@ class _GeneratorHelper {
         <String, FieldElement>{}, (map, field) {
       if (!field.isPublic) {
         unavailableReasons[field.name] = 'It is assigned to a private field.';
-      } else if (jsonKeyFor(field).ignore == true) {
+      } else if (jsonKeyFor(field).ignore) {
```

Comment:
hmm so its important now that anybody using the newest json_serializable changes there json_annotation dependency as well to have the minimum bound of 0.2.8... but there is no way to really communicate that.

I am not sure what the right solution is here?

Reply:
ack